packages/lib/fs-driver-node.ts

```diff
@@ -97,6 +97,7 @@ export default class FsDriverNode extends FsDriverBase {
 			};
 		} catch (error) {
 			if (error.code === 'ENOENT') return null;
+			if (error.code === 'ENAMETOOLONG') return null;
```

Comment:
I don't think this logic should be here. The way stats work is that it either returns the file info, or null if the file doesn't exist. But in this case it exists, except it cannot be processed. So whatever we need to solve it's definitely not here that the fix should be

Reply:
The PR name made this more confusing, my bad.

These links are not local resources. The way that importer works is by reading the HTML for links and trying to check if they exist locally or not. When it is a will be a very long link like, or a `mailto:` to a support address with a template included, for example, `fsDriver` `stat` will throw the `ENAMETOOLONG`.

There isn't any way to file exist because `ENAMETOOLONG` since it means it is a name longer than the system supports.